tests/functional_tests/test_scans_postgres.py

```diff
@@ -0,0 +1,57 @@
+import os
+import shutil
+import tempfile
+import unittest
+
+from credentialdigger.cli import cli
+from git import Repo as GitRepo
+from psycopg2 import connect
+
+
+class TestScansPostgres(unittest.TestCase):
+    dotenv = os.path.join(os.path.dirname(os.path.abspath(__file__)), ".env")
+    repo_url = 'https://github.com/fabiosangregorio/credential-digger-tests'
```

Comment:
we keep it here at the moment, but we will probably need to update this url. Nothing personal, just compliancy 😄